File: createJson/src/main/java/com/ruzhan/jsonfile/model/MovieDetail.java
package com.ruzhan.jsonfile.model;

import java.util.List;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class MovieDetail {

    public String id;
    public String movieId;
    public String title;
    public String tag;
    public String desc;
    public String image;

    public List<Introduce> introduces;
    public List<Video> videos;
}

File: createJson/src/main/java/com/ruzhan/jsonfile/model/MovieDetail.kt
package com.ruzhan.jsonfile.model

class MovieDetail {

    var id: String? = null
    var movieId: String? = null
    var title: String? = null
    var tag: String? = null
    var desc: String? = null
    var image: String? = null

    var introduces = ArrayList<Introduce>()
    var videos = ArrayList<Video>()
}
